app/views/play/CampaignView.coffee

```diff
@@ -754,7 +756,11 @@ module.exports = class CampaignView extends RootView
     levelURL = "/db/level/#{levelSlug}"
     level = new Level().setURL levelURL
     level = @supermodel.loadModel(level, null, 0).model
-    sessionURL = "/db/level/#{levelSlug}/session"
+
+    # Note that this not only preloads the level, but also creates it if the
+    # level requires the campaign to be included, which the game-dev-hoc premium levels.
```

Comment:
`Note this preloads the level, creates ?? if the level requires the campaign to be included.  E.g. game-dev-hoc premium levels.`

Reply:
Clarified comment:

```
    # Note that this doesn't just preload the level. For sessions which require the
    # campaign to be included, it also creates the session. If this code is changed,
    # make sure to accommodate campaigns with free-in-certain-campaign-contexts levels,
    # such as game dev levels in game-dev-hoc.
```